MockKitUI/View Controllers/IssueAlertTableViewController.swift

```diff
@@ -0,0 +1,111 @@
+//
+//  IssueAlertTableViewController.swift
+//  MockKitUI
+//
+//  Created by Rick Pasetto on 4/24/20.
+//  Copyright © 2020 LoopKit Authors. All rights reserved.
+//
+
+import UIKit
+import LoopKit
+import LoopKitUI
+import MockKit
+
+final class IssueAlertTableViewController: UITableViewController {
+  
+    let cgmManager: MockCGMManager
+
+    private enum AlertRow: Int, CaseIterable, CustomStringConvertible {
+        case immediate = 0
+        case delayed
+        case repeating
+        case issueLater
+        var description: String {
```

Comment:
[nit] Can you add a line break between the case declarations and the computed properties?